data-serving/data-service/src/controllers/case.ts

```diff
@@ -40,3 +40,21 @@ export const create = async (req: Request, res: Response) => {
 
     res.send(`Triggered create case with age: ${req.body.age}`);
 };
+
+/**
```

Comment:
Er, merge looks a little weird?

Reply:
I DON'T EVEN KNOW WHY THIS FILE IS EDITED PLEASE SEND HELP